### PR TITLE
Initialize outpatientConsult form

### PR DESCRIPTION
**Pull request by brandones**
Some config changes and some boilerplate forms.

### CODE REVIEW

File: configuration/pih/pih-config-peru.json [hunk 1]
```diff
@@ -30,30 +35,12 @@
...
-      },
-      "isDisabled": {
-        "required": false
-      },
-      "isImmigrant": {
-        "required": false
-      },
-      "isIndigenous": {
-        "required": false
-      }
```
Comment (brandones) [inline configuration/pih/pih-config-peru.json:56]: Son especificos a los secciones del registracion de Mexico

File: configuration/pih/pih-config-peru.json [hunk 2]
```diff
@@ -20,7 +25,7 @@
   "providerIdentifierPrefix": "P",
   "primaryIdentifierPrefix": "SES",
   "addressConfig": {
-    "manualFields": ["address1", "address2", "address3"],
+    "manualFields": ["address1"],
```
Comment (brandones) [inline configuration/pih/pih-config-peru.json:28]: Esta "addressConfig" configura cosas adicionales de addresshierarchy, que no estan configurado en `addressConfiguration.xml`. Olvidamos actualizarlos con el `addressConfiguration.xml`.

File: configuration/pih/pih-config-peru.json [hunk 3]
```diff
@@ -6,11 +6,16 @@
...
+    "conditionList",
+    "consult",
+    "hiv",
     "labs",
     "myAccount",
     "patientRegistration",
+    "programs",
     "systemAdministration",
     "visitManagement",
+    "visitNote",
```
Comment (brandones) [inline configuration/pih/pih-config-peru.json:18]: Por favor referir a https://github.com/PIH/openmrs-config-pihemr/#existing-components para ver que hacen estos

File: configuration/pih/pih-config-peru.json (hunk 1 above)
Comment (aastupilloses) [reply configuration/pih/pih-config-peru.json:56]: Hi, @brandones. I don't know if we would use one of these in the future, but it's OK for me to disable it for the moment.

File: configuration/pih/pih-config-peru.json (hunk 2 above)
Comment (aastupilloses) [reply configuration/pih/pih-config-peru.json:28]: Hi, @brandones. This is algo OK for me. We are not gonna use it.

File: configuration/pih/pih-config-peru.json (hunk 3 above)
Comment (aastupilloses) [reply configuration/pih/pih-config-peru.json:18]: Hi, @brandones. I know those components were the ones added in the "lesson" we had with you previous week. So, it's OK too. We are gonna use this.

**Comment by aastupilloses** [comment]
Hi, @brandones. After reviewing all the changes, I'm approving this and merging it into master. Thanks for your help!

**Pull request merged**